apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -355,13 +385,30 @@ class VisualizerModal extends React.Component {
                 onChange={event =>
                   this.setState({
                     filterColumn: event.target.value,
+                    filterOperator: '=',
                     filterValue: ''
                   })
                 }
                 inlineLabel
               />
+              {displayNumericFilters && (
+                <DropdownField
+                  displayName={msg.by()}
+                  options={['>', '>=', '=', '<=', '<']}
```

Comment:
I'm tempted to make these constants similar to how we do ColumnType or ChartType. It won't give us the benefit of making it simpler to add new values (since I can't image what other options there would be), but it will make it less prone to typos if we edit in this area and will follow the style pattern setup by Column and ChartType.